cg/store/models.py

```diff
@@ -991,9 +991,12 @@ class Order(Base):
     customer: Mapped[Customer] = orm.relationship(foreign_keys=[customer_id])
     order_date: Mapped[datetime] = mapped_column(default=datetime.now)
     ticket_id: Mapped[int] = mapped_column(unique=True, index=True)
-    workflow: Mapped[str] = mapped_column(types.Enum(*(workflow.value for workflow in Workflow)))
```

Comment:
I think the OrdersResponse in one of the endpoints (maybe more response DTOs as well) just renders the database entries. We might need to add the workflow to the response DTOs to not break functionality in the FE

Reply:
I think the property takes care of that. It was accessing orders.workflow (which now still is there)

Reply:
Oh, I got confused by the DatabaseOrder and the Order name clashes. You are most likely correct